pkg/pillar/types/zedmanagertypes.go

```diff
@@ -263,7 +263,8 @@ type AppInstanceStatus struct {
 	// Effective time, when the application should start
 	StartTime time.Time
 	// Snapshot related information
-	SnapStatus SnapshottingStatus
+	SnapStatus  SnapshottingStatus
+	MemOverhead uint64 // Overhead for memory for VM
```

Comment:
Please add the unit in the comment. (I guess it is MB.)

Reply:
Fixed